### DISCUSSION TITLE
ColumnLineChart 에서 zoomable 가능한가요?

### DISCUSSION THREAD

**Discussion by devjsy0897** [nhn/tui.chart#745] [Q&A]
안녕하세요.
ColumnLineChart 를 사용해 개발중에 있습니다.
Line Chart 같은 경우는 zoomable이 가능하다고 확인했는데 ColumnLine Chart는 적용 예시가 없어서 물어봅니다.
첨부사진과 같이 X축(시간데이터)가 많은 경우 확대 해서 볼 수있게 하는 방법은 어떤것이 있나요
![nhnchart](https://user-images.githubusercontent.com/69065695/170169226-690091c3-f2e6-4a58-9d93-a18f2f5cfdd9.png)

**Comment by jwlee1108** [comment]
@devjsy0897

Sorry for the late response. 
ColumnLine doesn't support it. because ColumnLine is combined with Column and Line. and Column Chart doesn't support `zoom` feature.
